charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -119,20 +104,21 @@ spec:
         - name: scalardb-cluster-database-properties-volume
           configMap:
             name: {{ include "scalardb-cluster.fullname" . }}-node-properties
-        {{- if .Values.scalardbCluster.tls.caRootCertSecret }}
-        - name: scalardb-cluster-tls-ca-root-volume
+        {{- if and .Values.scalardbCluster.tls.enabled .Values.scalardbCluster.tls.certManager.enabled }}
+        - name: scalardb-cluster-tls-volume
           secret:
-            secretName: {{ .Values.scalardbCluster.tls.caRootCertSecret }}
+            secretName: {{ include "scalardb-cluster.fullname" . }}-tls-cert
         {{- end }}
-        {{- if .Values.scalardbCluster.tls.certChainSecret }}
-        - name: scalardb-cluster-tls-cert-chain-volume
-          secret:
-            secretName: {{ .Values.scalardbCluster.tls.certChainSecret }}
-        {{- end }}
-        {{- if .Values.scalardbCluster.tls.privateKeySecret }}
-        - name: scalardb-cluster-tls-private-key-volume
-          secret:
-            secretName: {{ .Values.scalardbCluster.tls.privateKeySecret }}
+        {{- if and (.Values.scalardbCluster.tls.enabled) (not .Values.scalardbCluster.tls.certManager.enabled) }}
+        - name: scalardb-cluster-tls-volume
+          projected:
+            sources:
+              - secret:
+                  name: {{ .Values.scalardbCluster.tls.caRootCertSecret }}
+              - secret:
+                  name: {{ .Values.scalardbCluster.tls.certChainSecret }}
+              - secret:
+                  name: {{ .Values.scalardbCluster.tls.privateKeySecret }}
```

Comment:
If you don't use cert-manager (and you want to manage each file separately), you have to create three secrets for the private key, certificate, and CA certificate.

In this case, I want to combine them into one volume to mount them easily under the same directory in the container. So, I use `projected` volume here.